File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_2/exercice08.py
H=int(input("saisir le nombre des heures: "))
MIN=int(input("saisir le nombre des minutes: "))
if MIN<59 :
    MIN=MIN+1
elif MIN==59:
    MIN=0
    if H<23:
        H=H+1
    elif H==23:
        H=0
print(f"Dans une minute, il sera {H} heure(s) {MIN}")

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_2/exercice08.c
#include <stdio.h>
int H,MIN;
int main(){
    printf("saisir le nombre des heures : ");
    scanf("%d",&H);
    printf("saisir le nombres des minutes: ");
    scanf("%d",&MIN);
    if(MIN<59){
        MIN++;
    }
    else if(MIN==59){
        if(H<23){
            MIN=0;
            H++;
        }
        else if(H=23){
            MIN=0;
            H=0;
        }
    }
    printf("Dans une minute, il sera %d heure(s) %d",H,MIN);
    return 0;
}